brush-core/src/lib.rs

```diff
@@ -36,6 +36,7 @@ pub mod traps;
 pub mod variables;
 
 pub use arithmetic::EvalError;
+pub use brush_parser::ParseError;
```

Comment:
Apologies that this PR has gotten a bid wedged; let's try to find some initial step we can move forward with. Here's my incremental proposal for now:

```suggestion
/// Module containing the core parser facilities used and re-exported by this crate.
pub mod parser {
    pub use brush_parser::{
        ParseError, ParserOptions, SourcePosition, Token, TokenLocation, TokenizerError, ast,
    };
}
```

I think we can do better in the future, but I'm hoping this will give you a fair bit to go with. Do you think this will meet your needs for the time being, @hwittenborn?